src/plugin/action/join.sjs

```diff
@@ -104,10 +104,16 @@ module.exports = function (client, rawf, emitter) {
             };
 
             client.debug("PluginAction", formatc("Attempting to join %s."));
-            rawf("JOIN :%s", channel);
+            if(typeof(channel) === 'object'){
```

Comment:
Space after `if`.  Strings should be double quotes. Space before the opening bracket.